lib/colvars/Makefile.deps

```diff
@@ -1,9 +1,9 @@
 
 $(COLVARS_OBJ_DIR)colvaratoms.o: colvaratoms.cpp colvarmodule.h \
- colvars_version.h colvarproxy.h colvarvalue.h colvartypes.h \
+ colvars_version.h colvarproxy.h colvartypes.h colvarvalue.h \
```

Comment:
Commit does not change the dependencies, but just their order.

Reply:
i know. that is why i won't push them upstream. it is just convenient to me to have this consistent with my personal development environment, so it doesn't show up all the time as modified.